### ISSUE TITLE
Is there any way to get the list of users currently logged in / the history of logins?

### ISSUE THREAD

**Issue by gitime** [sscarduzio/elasticsearch-readonlyrest-plugin#141]
Hi,

I would like to monitor login history of readonlyrest. I have different users / groups. I would like to monitor currently logged in users. Or its okay if I get a history of user logins (in a file / elasticsearch index). If I get data in elasticsearch index, I hope I can view it in Kibana dashboard. Any way or workaround for this?

Thank you.

**Comment by sscarduzio** [comment]
Starting from the upcoming redonlyrest 1.13.0, the elasticsearch log file will now show what user the request is logged in as. 
I.e.
```
[2016-12-27T06:20:08,454][INFO ][o.e.p.r.a.ACL            ] request: { id: 1593eef9f86, type: SearchRequest, user: kibana, action: indices:data/read/search, OA:172.17.0.1, indices:.kibana, M:POST, P:/.kibana/config/_search, C:<OMITTED, LENGTH=77>, Headers:[Authorization=Basic a2liYW5hOmtpYmFuYQ==, content-type=application/json, Host=localhost:9200, Content-Length=77, Connection=keep-alive], History:::LOGSTASH::=[auth_key=false],::KIBANA-SRV::=[auth_key=true] } matched block: ::KIBANA-SRV:: match: true}
```

notice the `user: kibana` extracted from the Authorization header in the HTTP request.

You can use Logstash to monitor this file, pluck the bits you need and send them to an index in ES.

Would be nice though to have an internal index with all the audit. Thanks for the idea @gitime :)

**Comment by sscarduzio** [comment]
FYI: 1.13.0 will come out with important bug fixes and enhancements as soon as Kibana 5.1.2 is released. If interested, follow this: https://github.com/elastic/kibana/issues/9583
